src/RawFile.test.cpp

```diff
@@ -160,6 +163,116 @@ TEST_CASE("Read multipart files", "[.integration]") {
     }
 }
 
+// dummy test class for easy member access
+class RawFileTestWrapper : public RawFile {
```

Comment:
We should maybe have a longer discussion but I'm not sure this is the way to go. Do we need to test the implementation or is it enough to test the public interface?